src/System.Diagnostics.Tracing/src/System.Diagnostics.Tracing.csproj

```diff
@@ -11,7 +11,7 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetGroup)'=='netcore50aot'">
-    <DefineConstants>$(DefineConstants);PROJECTN</DefineConstants>
+    <DefineConstants>$(DefineConstants);PROJECTN;FEATURE_SERIALIZATION</DefineConstants>
```

Comment:
>FEATURE_SERIALIZATION [](start = 49, length = 21)

I see this used in EventSourceException and nowhere else in all of corefx. How come it's only needed in just this one place?